src/content/en/tools/workbox/guides/advanced-recipes.md

```diff
@@ -15,32 +15,42 @@ worker has updated and waiting to install.
 
 To do this you'll need to add some code to your page and to your service worker.
 
-**Add to your page**
+**Code in your page**
 
 ```html
 <script type="module">
-import {Workbox} from 'https://storage.googleapis.com/workbox-cdn/releases/4.0.0/workbox-window.prod.mjs';
+import {Workbox} from 'https://storage.googleapis.com/workbox-cdn/releases/4.1.0/workbox-window.prod.mjs';
 
 if ('serviceWorker' in navigator) {
   const wb = new Workbox('/sw.js');
 
   // Add an event listener to detect when the registered
   // service worker has installed but is waiting to activate.
-  wb.addEventListener('waiting', () => {
+  wb.addEventListener('waiting', (event) => {
+    // `event.wasWaitingBeforeRegister` will be true if this is
+    // the first time the updated service worker is waiting.
```

Comment:
Actually the reverse is true. `event.wasWaitingBeforeRegister` is true after a reload and it's *still* waiting.

Reply:
Gah. I knew that.